addon/scheduler.ts

```diff
@@ -146,37 +197,28 @@ export function _getScheduleFn(
 }
 
 export function _setCapabilities(newCapabilities = CAPABILITIES): void {
-  _capabilities = newCapabilities;
+  _capabilities = assign(_capabilities, newCapabilities);
```

Comment:
Do we really want to mutate the passed in pojo? I was thinking we would want:

```suggestion
  _capabilities = assign({}, _capabilities, newCapabilities);
```

Reply:
Yep, seems correct.